include/common_structs.h

```diff
@@ -1938,10 +1940,15 @@ typedef struct PauseMapSpace {
 } PauseMapSpace; // size = 0x14
 
 typedef struct MenuPanel {
-    /* 0x00 */ u8 initialized; //?
+    /* 0x00 */ union {
+    /*      */ s32 s;
+    /*      */ struct {
+    /* 0x00 */ u8 initialized;
     /* 0x01 */ s8 col;
     /* 0x02 */ s8 row;
     /* 0x03 */ u8 selected; // usually set to the current value from gridData
+                } c;
```

Comment:
The lack of indentation here feels kinda weird to me. Also on this particular line there's an extra space before the `{` :shipit: